scripts/bootstrap.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+err_report() {
+  echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+IFS=$'\n\t'
+
+function grant_cluster_admin {
+  local username="$1"
```

Comment:
this is not needed, since undeclared variables already guarded by `nounset`, and this is local scope to function anyway, no need to be that verbose to user